src/visitors/ast.rs

```diff
@@ -54,10 +24,8 @@ impl<'a> NodeFinderVisitor<'a> {
 
 impl<'a> walk::Visitor<'a> for NodeFinderVisitor<'a> {
     fn visit(&mut self, node: walk::Node<'a>) -> bool {
-        if crate::lsp::position_in_range(
-            &self.position,
-            &node.base().clone().location.into(),
-        ) {
+        let range = lsp::Range::from(node.base().clone().location);
```

Comment:
I mainly changed this because I had added a `dbg!(range, self.position)` when working through the test failures. Which I prefer "depends", either way works.